udata/tests/forms/test_tag_field.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from udata.tests import TestCase
+from udata.forms.fields import tags_list, slug
+
+
+class TagFieldTest(TestCase):
+
+    def test_tags_list(self):
+        self.assertEquals([], tags_list(''))
+        self.assertEquals(['a'], tags_list('a'))
+        self.assertEquals(['a', 'ac', 'b'],
+                          sorted(tags_list('a, b ,  ,,, a,  ac')))
+        self.assertEquals(['a-b', 'c', 'd-e'],
+                          sorted(tags_list('a b, c , d e')))
+
+    def test_slug(self):
+        self.assertEquals('', slug(''))
+        self.assertEquals('ab-c-d', slug('ab c  d '))
```

Comment:
I think we need more test case, in order to ensure the awesome slugify configuration match our expectations.
Some expected cases:
- accents slugification (ie. `é` becomes `e`) (may be subject to discussion, do we want to strip accents ?)
- case lowering (ie. `aBc` becomes `abc`)
- inside spaces deduplicate (ie `a  b` becomes `a-b` and not `a--b`)
- max length handling
- special chars treatment (do we keep `.`, `_`...)

To be, the following cases are subject to debate:
- do we keep inner spaces (`ecole-publique` vs. `ecole publique`) ?
- do we keep accents (`école` vs. `ecole`) ?

My position: all in or all out, ie. `école publique` or `ecole-publique`. But I can't chose between the two, both have pros and cons.